src/components/LinkPreview.tsx

```diff
@@ -0,0 +1,65 @@
+import { useEffect, useState } from "react";
+import styled from "styled-components";
+
+interface PropsUrl {
+  url: string;
+}
+
+const Wrapper = styled.div`
+  width: 300px;
```

Comment:
why magic 300px?

Reply:
this component will be used inside another components, so it shouldn't shouldn't have limited width